src/common/app-paths/directory-for-bundled-binaries/directory-for-bundled-binaries.injectable.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable, lifecycleEnum } from "@ogre-tools/injectable";
+import path from "path";
+import isDevelopmentInjectable from "../../vars/is-development.injectable";
+import contextDirInjectable from "../../vars/context-dir.injectable";
+
+const directoryForBundledBinariesInjectable = getInjectable({
+  instantiate: (di) => {
+    if (di.inject(isDevelopmentInjectable)) {
+      return path.join(di.inject(contextDirInjectable), "binaries");
+    }
+
+    return path.join(process.resourcesPath);
```

Comment:
```suggestion
    return path.join(process.resourcesPath, "binaries");
```
?

Reply:
We don't want to have `"binaries"` here, removed unnecessary `path.join`.